test/controllers/propertyLinking/FileUploadControllerSpec.scala

```diff
@@ -58,16 +57,6 @@ class FileUploadControllerSpec extends VoaPropertyLinkingSpec with FakeObjects{
       status(result) mustBe OK
     }
 
-  it should "return remove file success" in {
```

Comment:
why are we removing this test?

Reply:
same comment

Reply:
I can see now that that test has been replaced